test/pathod/test_pathod.py

```diff
@@ -66,7 +66,7 @@ def test_connect(self):
 class TestSSLCN(tutils.DaemonTests):
     ssl = True
     ssloptions = dict(
-        cn="foo.com"
+        cn=b"foo.com"
```

Comment:
The OpenSSL API expects bytes, so this change is correct by all means :smiley:
For UTF8 domains, IDNA encoding would be the way to go.